backend/open_webui/models/files.py

```
@@ -188,11 +215,35 @@ def get_files_by_user_id(self, user_id: str) -> list[FileModel]:
+
+                if form_data.data is not None:
+                    file.data = {**(file.data if file.data else {}), **form_data.data}
+
+                if form_data.meta is not None:
+                    file.meta = {**(file.meta if file.meta else {}), **form_data.meta}
+
+                file.updated_at = int(time.time())
+                db.commit()
+                return FileModel.model_validate(file)
+            except Exception as e:
+                log.exception(f"Error updating file completely by id: {e}")
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)